app/templates/discover.hbs

```diff
@@ -26,7 +26,7 @@
         </div>
     </div>
     {{#if queryBody}}
-        <div class="row" style="text-align: center; font-size:75%; margin-top:15px">
+        <div class="row hidden-xs" style="text-align: center; font-size:75%; margin-top:15px">
```

Comment:
I feel like we don't want to hide features from mobile users, though (now that we're acknowledging they exist...). Could the sidebar be expandable?

I guess they can filter by anything in the search bar if they know the syntax, though, so the feature is still technically available, if opaque.